hivemind/utils/grpc.py

```diff
@@ -231,6 +284,13 @@ def deserialize_torch_tensor(serialized_tensor: runtime_pb2.Tensor) -> torch.Ten
     elif serialized_tensor.compression == CompressionType.FLOAT16:
         array = np.frombuffer(serialized_tensor.buffer, dtype=np.float16).copy()
         tensor = torch.as_tensor(array, dtype=torch.float32).view(*serialized_tensor.size)
+    elif serialized_tensor.compression == CompressionType.QUANTILE_8BIT:
+        lookup = serialized_tensor.buffer[:256*4]
```

Comment:
We'd better extract this value of 256*4 to a variable or a constant, because right now its meaning is unclear

Reply:
something like QUANTILE_8BIT_LOOKUP_SIZE?